src/lib.rs

```diff
@@ -33,6 +33,7 @@ pub mod license;
 pub mod lockfile;
 pub mod manifest;
 pub mod npm;
+pub mod opt;
```

Comment:
Maybe name this module `wasm_opt`? I know that we name the `wasm-bindgen`-related module just `bindgen` but I think in that case it is obvious what `bindgen` we are talking about, where as `opt` is not obviously `optimize` (as opposed to `options`) to me, and even if it was obviously `optimize` it isn't also obviously the-specific-tool-`wasm-opt`(imize)-and-not-generic-optimization-passes to me.

Reply:
👍